{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2018-0502",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [

        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [

          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://bugs.debian.org/908000",
          "name": "https://bugs.debian.org/908000",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "https://sourceforge.net/p/zsh/code/ci/1c4c7b6a4d17294df028322b70c53803a402233d",
          "name": "https://sourceforge.net/p/zsh/code/ci/1c4c7b6a4d17294df028322b70c53803a402233d",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "https://usn.ubuntu.com/3764-1/",
          "name": "USN-3764-1",
          "refsource": "UBUNTU",
          "tags": [

          ]
        },
        {
          "url": "https://www.zsh.org/mla/zsh-announce/136",
          "name": "https://www.zsh.org/mla/zsh-announce/136",
          "refsource": "MISC",
          "tags": [

          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "An issue was discovered in zsh before 5.6. The beginning of a #! script file was mishandled, potentially leading to an execve call to a program named on the second line."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [

    ]
  },
  "impact": {
  },
  "publishedDate": "2018-09-05T08:29Z",
  "lastModifiedDate": "2018-09-12T10:29Z"
}